openwisp_radius/tests/test_api.py

```diff
@@ -2594,16 +2595,21 @@ def test_register_201_phone_number_empty(self):
         self.assertEqual(User.objects.count(), 1)
         url = reverse('radius:rest_register', args=[self.default_org.slug])
         phone_number = '+393664255801'
-        r = self.client.post(
-            url,
-            {
-                'username': self._test_email,
-                'email': self._test_email,
-                'password1': 'password',
-                'password2': 'password',
-                'phone_number': phone_number,
-            },
-        )
+        numQueries = 50
+        # TODO: Remove this when support for django 2 is dropped
+        if django.get_version().startswith('2'):
```

Comment:
this is an unusual way to do the comparison, I suggest to keep consistency with the way it's usually done, eg:
https://github.com/openwisp/openwisp-users/blob/fabef8d63912c0c492248079e82fb3092f9b8ab8/openwisp_users/base/models.py#L166

There are many ways of doing it, but that's the one of the simplest and the one we've been using in several ow modules.

Reply:
Done :+1: